--- binutils-2.14/gas/config/tc-m68k.h	2003-01-23 06:51:05.000000000 -0600
+++ binutils-2.14/gas/config/tc-m68k.h	2026-02-21 15:03:28.858523054 -0600
@@ -206,8 +206,8 @@
 #define TARGET_WORD_SIZE 32
 #define TARGET_ARCH bfd_arch_m68k
 
-extern struct relax_type md_relax_table[];
-#define TC_GENERIC_RELAX_TABLE md_relax_table
+// extern struct relax_type md_relax_table[];
+// #define TC_GENERIC_RELAX_TABLE md_relax_table
 
 /* We can't do a byte jump to the next instruction, so in that case
    force word mode by faking AIM.  */
